flink-runtime/src/test/java/org/apache/flink/runtime/operators/DataSinkTaskTest.java

```diff
@@ -49,22 +50,17 @@
 import static org.junit.Assert.assertTrue;
 
 public class DataSinkTaskTest extends TaskTestBase {
+
+	@Rule
+	public TemporaryFolder tempFolder = new TemporaryFolder();
 	
 	private static final Logger LOG = LoggerFactory.getLogger(DataSinkTaskTest.class);
 
 	private static final int MEMORY_MANAGER_SIZE = 3 * 1024 * 1024;
 
 	private static final int NETWORK_BUFFER_SIZE = 1024;
 
-	private final String tempTestPath = constructTestPath(DataSinkTaskTest.class, "dst_test");
-
-	@After
-	public void cleanUp() {
-		File tempTestFile = new File(this.tempTestPath);
-		if(tempTestFile.exists()) {
-			tempTestFile.delete();
-		}
-	}
+	private final String tempTestFileName = getClass().getName() + "-dst_test";
```

Comment:
since this was only used to prevent name clashes between tests it should now be redundant and can be removed.